src/mongocxx/private/client.hpp

```diff
@@ -33,6 +33,7 @@ class client::impl {
     }
 
     mongoc_client_t* client_t;
+    std::vector<bsoncxx::string::view_or_value> ssl_options;
```

Comment:
@Machyne. Its just a vector :) I think it is OK to say it more than once. In fact, it can be annoying to need to chase down typedefs for every vector.